### PR TITLE
WIP - Whitelabel migration guide

### PR DESCRIPTION
**Pull request by flovntp**
## Why
We need a whitelabel migration guide to explain how to migrate config files manually or using whitelabel-ify

Closes #{ISSUE_NUMBER}

### CODE REVIEW

File: sites/friday/src/migration/from_platformsh.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+  disk: 1024
+```
+
+{{< note >}}
+These settings, size, disk and resources are no longer available in the Upsun Yaml config files (or not taken in account anymore).
+{{< /note >}}
+
+### Using the API
+#### Change app resources
+To change your `app` application resources, use the following command line:
```
Comment (gilzow) [inline sites/friday/src/migration/from_platformsh.md:220]: Per OVV, in a first build instance, if more than one service needs to be configured, you have to do both at once. You can't do the app, and then the database. We'll need to either update this section or, at a minimum, a warning/note.

File: sites/friday/src/migration/from_platformsh.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+
+### Using CLI (TODO)
+TODO
+### Using Console (TODO)
+TODO
+
+## Deploy your Upsun project
+After adding to Git your changes, use the following command to deploy your Upsun project:
+```shell
+upsun deploy
```
Comment (gilzow) [inline sites/friday/src/migration/from_platformsh.md:266]: are we sure the upsun cli has a `deploy` command? Latest version (4.2.1, Build date: 2023-08-09T10:22:55+00:00) does not contain it.

**Comment by chadwcarlson** [comment]
Closing for now, to reopen against new file structure.

**Comment by chadwcarlson** [comment]
Reopened in https://github.com/platformsh/platformsh-docs/pull/3351

**Pull request closed without merging**